deliverables/README.md

```diff
@@ -31,38 +38,116 @@ What is the role of Javascript in a web page?
+- include from file with ```<link rel="stylesheet" href="/path/to/stylesheet">```
+- inline in the tag with the style attribute like ```<p style="color: red;">asdf</p>```
+- between ```<style></style>``` tags
 
 ---
 ### Section 3: Git and Github  
 What is Git?  
+Git is a piece of version control software used for managing a code base.
+
 What is SCM?  
+An SCM is either
+	- Source Control Management: just the software involved with versioning and revisions in a codebase, like git
+	- Software Configuration Management: software package involving versioning, distribution, dependencies, etc
+
 What is a VCS?  
+Version control software, a tool to manage revisions and backups of a codebase.
+
 Why is Git useful for a developer?  
+It allows developers to both track progress and revert bad changes easily, as well as  
+compartmentalization of new features  
+
 Why is Git useful for a team of developers?  
+The development cycle and features can be tracked easier with git, and one can isolate different
+sections of a project with branches similar to if it were just a single developer. 
+
 How do you create a new Git repository for a project locally?  
+git init
+
 How do you create it on Github?  
+Click new repository > enter repo name and description 
+
 How do you commit changes?  
+1. First add files to commit with ```git add <files>```  
+2. commit the changes with git commit -m "commit message goes here"  
+
 What is the difference between staging and committing changes?  
+"git add" stages files for a commit, "git commit" creates a new commit that 'saves' the staged files  
+
 What is the difference between committing your changes and pushing them to Github?  
+committing saves the changes locally, pushing uploads the code to the remote repository (in this case github)  
+
 What is the command to check the status of your current repo in git?  
+git status  
+
 What is the command to see the history of your previous commits (from the command line)?  What are a few interesting variations (sets of options)?  
+```  
+git log  
+ -n <#> : limits commits displayed to the # given  
+ -until=<date> : shows commits older than date  
+ -since=<date> : shows commmits newer than date  
+ -author=<pattern> : matches regex pattern with commit author  
+ ```
+
 How can you look through your historical commits on the Github website?  
+click the "<#> commits" button that says how many commits the repo has, it's below  
+the repo description
+
 What is a "Merge"?  
+a merge is combining two different versions of a codebase/files in a codebase  
+
 What is a "Pull Request"?  
+a pull request is a request to a repository to have one's changes pulled into the main codebase  
+
 What is "Forking" a repo?  
+forking is copying a repo into one's own account  
+
 What is "Cloning" a repo?  
+cloning a repo is copying a repo from a remote repository to a local version  
```

Comment:
"clone" is usually from remote to local, but technically it can be from one directory to another on a single computer.